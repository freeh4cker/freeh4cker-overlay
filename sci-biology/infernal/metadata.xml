<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
<pkgmetadata>
	<herd>sci-biology</herd>
  <maintainer type="project">
    <email>sci-biology@gentoo.org</email>
    <name>Gentoo Biology Project</name>
  </maintainer>
  <longdescription>
Infernal ("INFERence of RNA ALignment") is for 
searching DNA sequence databases for RNA structure
and sequence similarities.

It is an implementation of a special case of 
profile stochastic context-free grammars called 
covariance models (CMs)
.
A CM is like a sequence profile, but it scores a
combination of sequence consensus and RNA 
secondary structure consensus, so in many cases,
it is more capable of identifying RNA homologs 
that conserve their secondary structure more 
than their primary sequence.
  </longdescription>
</pkgmetadata>
