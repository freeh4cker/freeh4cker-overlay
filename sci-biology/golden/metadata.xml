<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
<pkgmetadata>
  <herd>sci-biology</herd>
  <maintainer>
    <email>freeh4cker@gmail.com</email>
    <name>Bertrand Néron</name>
  </maintainer>
  <longdescription>
Golden is a small databanks entries retriever. It searches and displays
entries on indexed databanks 'flat' files.

This package currently provide the following programs 
	golden ... Entries retriever.
	goldin ... Indexes generator

Queries/Indexes can be made using entry name, primary accession numbers
or even both (by default)

Currently supported formats: Genbank, Embl, Pir codata.
  </longdescription>
  <upstream>
    <maintainer status="active">
	     <email>vlegrand@pasteur.fr</email>	
		   <name>Véronique Legrand</name>
		</maintainer>
		<bugs-to>velegrand@pasteur.fr</bugs-to>
  </upstream>
</pkgmetadata>
